jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MojoCommon.java

```diff
@@ -76,18 +76,21 @@ static void finishUpdateChecker(
       ProjectProperties projectProperties, Future<Optional<String>> updateCheckFuture) {
     UpdateChecker.finishUpdateCheck(updateCheckFuture)
         .ifPresent(
-            updateMessage -> {
-              projectProperties.log(LogEvent.lifecycle(""));
-              projectProperties.log(LogEvent.lifecycle("\u001B[33m" + updateMessage + "\u001B[0m"));
-              projectProperties.log(
-                  LogEvent.lifecycle(
-                      "\u001B[33m"
-                          + ProjectInfo.GITHUB_URL
-                          + "/blob/master/jib-maven-plugin/CHANGELOG.md\u001B[0m"));
-              projectProperties.log(
-                  LogEvent.lifecycle(
-                      "Please see https://github.com/GoogleContainerTools/jib/blob/master/docs/privacy.md for info on disabling this update check."));
-              projectProperties.log(LogEvent.lifecycle(""));
+            latestVersion -> {
+              String changelogUrl =
+                  ProjectInfo.GITHUB_URL + "/blob/master/jib-maven-plugin/CHANGELOG.md";
+              String privacyUrl = ProjectInfo.GITHUB_URL + "/blob/master/docs/privacy.md";
+              String message =
+                  String.format(
+                      "\n\u001B[33mA new version of %s (%s) is available (currently using %s). Update your"
```

Comment:
Sorry, missed this. For Maven, we intentionally use multiple `.log()` instead of `\n` when we need to start a new line. Not a big deal, but this is because Maven prepends the verbosity info on every line, for example,
```
[INFO] Containerizing application to francium25/test...
```